src/test/java/wooteco/subway/web/favorite/FavoriteControllerIntegrationTest.java

```diff
@@ -0,0 +1,236 @@
+                .header("Authorization", "Bearer" + unExistEmailToken)
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isUnauthorized());
+    }
+
+    @DisplayName("즐겨찾기 삭제가 성공한다")
+    @Test
+    void deleteFavoriteSuccessTest() throws Exception {
+        //given
+        CreateFavoriteRequest favorite1 = new CreateFavoriteRequest("강남역", "양재역", DummyTestUserInfo.EMAIL);
+        Favorite present = favoriteService.save(favorite1, DummyTestUserInfo.EMAIL);
+        Member member = new Member(DummyTestUserInfo.EMAIL, DummyTestUserInfo.NAME, DummyTestUserInfo.PASSWORD);
+        memberService.createMember(member);
+
+        String token = jwtTokenProvider.createToken(member.getEmail());
+        String uri = "/favorites/" + present.getId();
+
+        //when
+        mockMvc.perform(RestDocumentationRequestBuilders.delete("/favorites/{id}", present.getId())
+                .header("Authorization", "Bearer " + token)
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(FavoriteDocumentation.delete())
+                .andDo(print())
+                .andExpect(status().isNoContent());
+    }
+
+    @DisplayName("토큰이 없는 상태에서 즐겨찾기 삭제가 실패한다")
+    @Test
+    void deleteFavoriteFailTest() throws Exception {
+        CreateFavoriteRequest favorite1 = new CreateFavoriteRequest("강남역", "양재역", DummyTestUserInfo.EMAIL);
+        Member member = new Member(DummyTestUserInfo.EMAIL, DummyTestUserInfo.NAME, DummyTestUserInfo.PASSWORD);
+
+        Favorite present = favoriteService.save(favorite1, favorite1.getEmail());
+        memberService.createMember(member);
+        favoriteService.save(favorite1, member.getEmail());
+
+        String uri = "/favorites/" + present.getId();
+
+        mockMvc.perform(delete(uri)
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isUnauthorized());
+    }
+
+    @DisplayName("회원이 아닌 이메일 JWT 토큰 헤더 요청 시 즐겨찾기 삭제 요청이 실패 한다")
+    @Test
+    void deleteFavoriteFailTest2() throws Exception {
+        CreateFavoriteRequest favorite1 = new CreateFavoriteRequest("강남역", "양재역", DummyTestUserInfo.EMAIL);
+        Member member = new Member(DummyTestUserInfo.EMAIL, DummyTestUserInfo.NAME, DummyTestUserInfo.PASSWORD);
+
+        Favorite present = favoriteService.save(favorite1, DummyTestUserInfo.EMAIL);
+        memberService.createMember(member);
+        favoriteService.save(favorite1, member.getEmail());
+
+        String unExistEmailToken = jwtTokenProvider.createToken("wrong@gmail.com");
+        String uri = "/favorites/?source=" + present.getSource() + "&target=" + present.getTarget();
+
+        mockMvc.perform(get(uri)
+                .header("Authorization", "Bearer" + unExistEmailToken)
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isUnauthorized());
```

Comment:
```suggestion
                .andExpect(status().isForbidden());
```

Unauthorized(401)와 Forbidden(403)의 차이를 공부해보세요!

Reply:
401은 익명의 사용자(로그인 X), 403는 로그인은 하였으나 권한이 없는 사용자가 되겠군요!